platform/darwin/src/MGLShapeSource.h

```diff
@@ -143,6 +142,41 @@ extern const MGLShapeSourceOption MGLShapeSourceOptionSimplificationTolerance;
  */
```

Comment:
This was done in https://github.com/mapbox/mapbox-gl-native/pull/7665/commits/6ba0034aea6ab709763d83bd138531ac31ddbe5a